### PR TITLE
New env var for aat env to get to CCD_ADMIN_URL

### PR DESCRIPTION
**Pull request by yarpo**
Trying to fix master.

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -64,5 +64,6 @@ withPipeline("nodejs", product, component) {
 
   after('functionalTest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test/functional/output/*'
+    env.CCD_ADMIN_URL="https://ccd-admin-web.aat.platform.hmcts.net"
```
Comment (shaunganley) [inline Jenkinsfile_CNP:67]: you will need to separate out `preview` and `aat` steps, this is the correct URL for AAT, not preview

File: charts/div-ccd-definitions/values.aat.template.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+tags:
+  ccd-idam-pr: false
+
+nodejs:
+  # Don't modify below here
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+steps_file:
```
Comment (shaunganley) [inline charts/div-ccd-definitions/values.aat.template.yaml:8]: this isn't required

File: Jenkinsfile_CNP (hunk 1 above)
Comment (yarpo) [reply Jenkinsfile_CNP:67]: yes, you're right

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -60,9 +60,15 @@ withPipeline("nodejs", product, component) {
...
     yarnBuilder.yarn('generate-bulk-excel-all')
+    env.CCD_ADMIN_URL="https://admin-web-div-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
   }
 
   after('functionalTest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test/functional/output/*'
   }
+
+  before('functionalTest:aat') {
+    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test/functional/output/*'
```
Comment (shaunganley) [inline Jenkinsfile_CNP:71]: this should be archiving the XLSX, not the test output, also add this to the before of preview if your want the configs to be archived

**Pull request merged**